src/install/requirements_pre_install.txt

```diff
@@ -4,5 +4,9 @@ virtualenv
 distro==1.7.0
 python-magic==0.4.25
 requests==2.27.1
+# Needed by config.py
+pydantic==1.8.2
```

Comment:
Hmm I'm not sure about that.
In my mind these the `requirements-{common,backend,frontend}.txt` always were one thing and `requirements_pre_install.txt` was seperate from that.

From my understanding the "pre_install" requirements are the dependencies of the installer. So the backend and frontend cant assume that these things are installed. If that is the case we should rename it to sth. like `requirements-installer.txt` and also leave pydantic in `requirements_common.txt`.

Reply:
> From my understanding the "pre_install" requirements are the dependencies of the installer.  

That is correct. But it's not like we uninstall them after installation, so we can assume that they are installed for the rest of the code. But it is probably better to also leave it in the `requirements_common.txt` since it is also used outside of the installation to avoid confusion even if that means some duplication. We should try to better keep track where the things in the installation files come from in general (sadly we haven't always done so in the past).